atlas/src/shared/components/VideoPreview/VideoPreview.tsx

```diff
@@ -71,6 +72,25 @@ const VideoPreview: React.FC<VideoPreviewProps> = ({
     onClick(e)
   }
 
+  const MIN_COVER_WIDTH = 300
```

Comment:
Where does this and `MAX_COVER_WIDTH` come from? Should this one by correlated with the default value of `Grid` prop `minWidth`? I'm afraid of having constant values all over the place. Also, how does `MAX_COVER_WIDTH` correlate with `MAX_VIDEO_PREVIEW_WIDTH` from `VideoPreviewBase.styles.tsx`? My guess would be that we only need one of them

Reply:
This choice for `MAX_COVER_WIDTH` is arbitrary. I set it to `600` as I've notice that previews on smaller screen almost never get wider than 600px so I've used to test how the Avatar and Title would look like in that case.
Note that, when previews get wider than 600px, for example on a _very_ wide screen,  the `scalingFactor` will keep increasing past `1.375` and so Title and Avatar will get bigger. 
I think `MAX_VIDEO_PREVIEW_WIDTH` in `VideoPreviewBase.styles.tsx` is stale code from when we had a `max-width` for video previews, and it was indeed that value. I don't think its used anywhere now.